src/config.rs

```diff
@@ -68,4 +73,12 @@ impl Config {
     fn default_double_click_delay() -> u64 {
         200
     }
+
+    fn default_error_format() -> String {
+        "$short_error_message|X".into()
```

Comment:
Can you update the table in the README too? For the others as well.

Reply:
Yeah, I was about to comment that I'm not sure where to document it. Like, there should also be a table of available placeholders and so on. And README is getting a bit cluttered...

Reply:
We could just list it in table for now and figure out how to improve it later.

Reply:
Okay